src/styles/card.scss

```diff
@@ -0,0 +1,94 @@
+@import 'variables';
+
+.card {
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: $white;
+  width: 198px;
+  height: 406px;
+&__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 35px;
+}
+&__text{
+  color: $color;
+  font-family: $font;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  width: 166px;
+  margin:1px 16px;
```

Comment:
reset margin for all tags you need and try to use only margin-bottom to get vertical indent between elements